### ISSUE TITLE
Wrong labels in the GAN example "Writing a training loop from scratch" 

### ISSUE THREAD

**Issue by Bekci** [keras-team/keras-io#361]
In the last part of the example (End-to-end example: a GAN training loop from scratch) should not we use 1 for the real examples and 0 for the fake examples.  

It creates labels for the [discriminator](https://github.com/keras-team/keras-io/blob/c9814a79182c1977ba28944174ad24dbcd517842/guides/writing_a_training_loop_from_scratch.py#L455 ) as 
```
    labels = tf.concat(
        [tf.ones((batch_size, 1)), tf.zeros((real_images.shape[0], 1))], axis=0
    )
```
and for the generator 
`misleading_labels = tf.zeros((batch_size, 1))`

I believe these should be:
```
    labels = tf.concat(
        [tf.zeros((batch_size, 1)), tf.ones((real_images.shape[0], 1))], axis=0
    )
```
and 
`misleading_labels = tf.ones((batch_size, 1))`

**Comment by eduman23** [comment]
Bekci, you are right, it think as same with you. it should be an inadvertently error while coding. 

because the output of discriminator is 1 for real and 0 for fake images. as coded here reversed.

**Comment by sachinprasadhs** [comment]
Hi, 

Please refer the comment in the linked PR and feel free to close the issue. Thanks!
